### DISCUSSION TITLE
[Question] Packet analysis through Wireshark

### DISCUSSION THREAD

**Discussion by ehgus607** [eProsima/Fast-DDS#4243] [Q&A]
Hello. I wanted to observe RTPS packets (e.g., ACK, HEARTBEAT, etc), so I checked the packets through wireshark for HelloWorld example. But I only observed INFO_TS packets. I attached the packets I captured. What I wanted to check is as follows:
1. Participant and Endpoint Discovery
I wanted to check the packets in the discovery phase, so I expected packets such as HEARTBEAT and ACKNACK, but I didn't observe them.

2. Transfer data
I thought that the data that publishers write in the topic is delivered in a packet. Therefore, I expected a DATA packet containing "Hello World" but I also didn't observe it.

I'm using the HelloWorld example as it is. Is there anything I need to customize? I'd appreciate it if you could help me.

Subscriber:
```Subscriber matched.
Message HelloWorld 1 RECEIVED
Message HelloWorld 2 RECEIVED
Message HelloWorld 3 RECEIVED
Message HelloWorld 4 RECEIVED
Message HelloWorld 5 RECEIVED
Message HelloWorld 6 RECEIVED
Message HelloWorld 7 RECEIVED
Message HelloWorld 8 RECEIVED
Message HelloWorld 9 RECEIVED
Message HelloWorld 10 RECEIVED
Subscriber unmatched.
```

Publisher:Publisher matched.
```Message: HelloWorld with index: 1 SENT
Message: HelloWorld with index: 2 SENT
Message: HelloWorld with index: 3 SENT
Message: HelloWorld with index: 4 SENT
Message: HelloWorld with index: 5 SENT
Message: HelloWorld with index: 6 SENT
Message: HelloWorld with index: 7 SENT
Message: HelloWorld with index: 8 SENT
Message: HelloWorld with index: 9 SENT
Message: HelloWorld with index: 10 SENT
```

Wireshark:
![Screenshot from 2024-01-13 15-49-29](https://github.com/eProsima/Fast-DDS/assets/20153878/0a097251-96b2-4aca-8ed1-3af1188f9514)

**Comment by JesusPoderoso** [comment]
Hi @ehgus607, thanks for using Fast DDS.
Please, check if [this thread answer](https://github.com/eProsima/Fast-DDS/discussions/4252#discussioncomment-8167663) solves your problem.

**Reply by ehgus607** [reply]
Thank you for response. I solved my problem now. I didn't know that SHM_TRANSPORT option is ON in default.

I will remain this disscussion for users trying wireshark.
